rainbow-me/browser-extension
# feat: token search (Cmd+K)

Fixes BX-1452, BX-1479
Figma link (if any):

## What changed (plus any additional context for devs)

With Cmd+K you can now paste in any token address and search for a token.

## Screen recordings / screenshots

https://github.com/rainbow-me/browser-extension/assets/53529533/94f863c3-9145-4584-9f34-a6435107d8ed

## What to test

- Open Cmd+K and paste in token address from any supported chains, ideally it would be best if you can test multiple token addresses from different chains. After you've done that you should be able to see them in Cmd+K menu.
- If you search a token address and you own the same token you should only see one token row in Cmd+K menu, which is your owned one.
- You cannot watch a token address as a wallet. You shouldn't see this:
<img width="348" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/53529533/ec0c458e-9824-498e-8517-d5412b2f7242">

- Make sure whenever you press "Enter" for a searched remote token it should redirect you to the token details page like you normally would with your owned tokens.
- The token details page shouldn't allow you to pin token, hide token and show you current balance and value.
- Make sure token details page for remote tokens works smoothly
